docs/docs/recipes.md

```diff
@@ -226,6 +227,46 @@ Data sourcing in Gatsby is plugin-driven; Source plugins fetch data from their s
 - Search available source plugins in the [Gatsby library](/plugins/?=source)
 - Understand source plugins by building one in the [Pixabay source plugin tutorial](/docs/pixabay-source-plugin-tutorial/)
 
+## Sourcing Content
+
+### Directly importing data (json & yaml)
+
+#### Prerequisites
+
+- A Gatsby site with two page components `index.js` and `page-2.js`
+- A valid YAML file and a valid JSON file inside a folder on your Gatsby website.
+
+#### Directions
+
+1. To import YAML content directly into a page. Open the index.js page component (src/pages/index.js), import the YAML file into the component as a default export.
+
+```jsx:title=src/pages/index.js
+import React from "react"
+import YAMLContent from "your-folder/your-yaml-file.yaml"
+
+export default () => (
+  <div style={{ color: `purple` }}>
+    <p>YAMLContent.the-property-you-want-to-display</p>
+  </div>
+)
+```
+
+2. To import JSON content directly into a page, open the page-2.js page component (src/pages/page-2.js) import the JSON file into the component as a default export.
```

Comment:
```suggestion
#### JSON Directions

1. Open the component in `src/pages/index.js`, import the JSON file, and assign it to a variable.

2. In the JSX markup, reference the variable and data structure returned from the JSON file to display it.
```
By listing the two options as separate and optional steps, we can eliminate the `page-2` requirement.